buildtools/pip/pip.go

```diff
@@ -35,26 +37,42 @@ func (p *Pip) Install(requirementsFilename string) error {
 	return err
 }
 
-func (p *Pip) List() ([]Requirement, error) {
+func (p *Pip) List() ([]Requirement, *errors.Error) {
 	stdout, _, err := exec.Run(exec.Cmd{
 		Name: p.Cmd,
 		Argv: []string{"list", "--format=json"},
 	})
 	if err != nil {
-		return nil, err
+		return nil, &errors.Error{
+			Cause:           err,
+			Type:            "shell",
+			Troubleshooting: fmt.Sprintf("Ensure that %s is installed, if it is then try to run %s list --format=json", p.Cmd, p.Cmd),
+			Link:            "https://github.com/fossas/fossa-cli/blob/master/docs/integrations/python.md#strategy-string",
+		}
 	}
 	var reqs []Requirement
 	err = json.Unmarshal([]byte(stdout), &reqs)
 	if err != nil {
-		return nil, err
+		return nil, &errors.Error{
+			Cause:           err,
+			Type:            "shell",
+			Troubleshooting: fmt.Sprintf("the following output from the command %s list --format=json could not be un-marshalled:\n%s\ntry running the command on your own and checking for any errors", p.Cmd, string(stdout)),
```

Comment:
For the Troubleshooting message, let's consistently capitalize the first letter please.